app/src/app/app.component.html

```diff
@@ -1,2 +1,2 @@
-<h1 class="sr-only">{{ pageHeading | translate }}</h1>
+<h1 class="sr-only" title="heading">{{ pageHeading | translate }}</h1>
```

Comment:
title should be localiseable

Reply:
@DenysVuika localization changes have been done now.